Makefile

```diff
@@ -43,7 +43,7 @@ refresh: | down build up ## Recreates local docker environment
 
 up: ## Runs the local containers (n=service name)
 	@echo "$(P) Running client and server..."
-	@docker-compose up -d $(n)
+	@docker-compose --env-file .env up -d $(n)
```

Comment:
Technically by default `docker-compose` will automatically look for a `.env` file in the same directory, but I've hardcoded it to ensure the user adds the file.

```conf
KEYCLOAK_PORT=8080
DATABASE_PORT=5433
API_HTTP_PORT=5000
API_HTTPS_PORT=5001
APP_HTTP_PORT=3000
```